pykubectl/kubectl.py

```diff
@@ -32,7 +35,7 @@ def delete(self, *args, **kwargs):
 
     def get(self, *args, **kwargs):
         result = self.execute('get -o json', *args, **kwargs).decode()
-        return json.loads(result)['items']
```

Comment:
Loading from file results in different one item being returned, where previously it was an array of "items"

Reply:
weird